RELEASE.md

```diff
@@ -1,78 +1,234 @@
 
-    * [test-runner](https://github.com/kubernetes/ingress-nginx/tree/main/images/echo)
-    * [echo](https://github.com/kubernetes/ingress-nginx/tree/main/images/echo)
-    * [cfssl](https://github.com/kubernetes/ingress-nginx/tree/main/images/cfssl)
-    * [fastcgi-helloserver](https://github.com/kubernetes/ingress-nginx/tree/main/images/fastcgi-helloserver)
-    * [httpbin](https://github.com/kubernetes/ingress-nginx/tree/main/images/httpbin)
+    - That single line formatted with "\n" needs to be changed as it does not meet yaml requirements
 
-* Open pull request
-* Merge
-* Wait for [cloud build](https://console.cloud.google.com/cloud-build/builds?project=k8s-staging-ingress-nginx)
+    - At the time of writing this doc, the 'configMap.data' spec is at line number 39.
 
-2a. Promote images:
+    - So editing begins at line 40 (at the time of writing this doc)
 
-* Open pull request to promote [staging image](https://github.com/kubernetes/k8s.io/blob/main/k8s.gcr.io/images/k8s-staging-ingress-nginx/images.yaml)
-    * e2e-test-runner
-    * e2e-test-cfssl
-    * e2e-test-echo
-    * e2e-test-fastcgi-helloserver
-    * e2e-test-httpbin
+    - Make that snippet look like this ;
+      ```
+      data:
+        http-snippet:|
+          server{
+          listen 2443;
+          return 308 https://$host$request_uri;
+          }
+      ```
 
-3. Update references to e2e-test-runner image:
+### f. Edit the changelog
+  [Changelog.md](https://github.com/kubernetes/ingress-nginx/blob/main/Changelog.md) 
+    - Add the PRs merged after previous release
 
-* [e2e-image](https://github.com/kubernetes/ingress-nginx/blob/main/test/e2e-image/Dockerfile#L1)
-* [run-in-docker.sh](https://github.com/kubernetes/ingress-nginx/blob/ff60aa9e2b5377db1544091b98f475a90a630297/build/run-in-docker.sh#L37)
+### g. Edit the Documentation:
+    - Update the version in [docs/deploy/index.md](docs/deploy/index.md)
+    - Update Supported versions in the Support Versions table in the README.md 
 
-4. Prepare for a new release:
+### h. Edit stable.txt
 
-* Change [TAG](https://github.com/kubernetes/ingress-nginx/blob/main/TAG#L1)
-* Open pull request
-* Merge
-* [Wait for cloud build](https://console.cloud.google.com/cloud-build/builds?project=k8s-staging-ingress-nginx)
+- Edit the stable.txt file(if applicable), in the root of the repo, to reflect the release to be created
 
-4a. Promote images:
+### i. Edit ghpages
 
-* Open pull request to promote [staging image](https://github.com/kubernetes/k8s.io/blob/main/k8s.gcr.io/images/k8s-staging-ingress-nginx/images.yaml)
-  * controller
+- Edit the ghpages file as needed
 
-5. Release helm chart:
+## 5. RELEASE new version
 
-* Open pull request updating [Chart.yaml](https://github.com/kubernetes/ingress-nginx/blob/main/charts/ingress-nginx/Chart.yaml#L3-L4)
-* Merge
-* [New helm chart is available](https://github.com/kubernetes/ingress-nginx/blob/main/.github/workflows/main.yaml#L47-L68)
+### a. Create PR
```

Comment:
There is no PR for the release, the release is a Github tool in the UI. At this point we have the tag, sha, updated docs with a changelog.

Reply:
unless we are still missing the Github Release?

Reply:
That is all gone. Wonder how you are seeing text from earlier.